sequences/README.md

```diff
@@ -0,0 +1,257 @@
+# Request Sequences
+
+Imagine, there are two micro-services, and we want to pass to the second service data from the response of the first. Using Couper, we can connect both services, using an implicit sequence.
```

Comment:
more rationale? a sequence implemented in the GW can decrease coupling between microservices. they don't need to "know" (or reach) each other. less configuration in the services.